app/user.go

```diff
@@ -1853,3 +1871,69 @@ func (a *App) UpdateOAuthUserAttrs(userData io.Reader, user *model.User, provide
 
 	return nil
 }
+
+// FilterNonTeamGroupMembers returns the subset of the given user IDs of the users who are not members of groups
+// associated to the team.
+func (a *App) FilterNonTeamGroupMembers(userIDs []string, team *model.Team) ([]string, error) {
```

Comment:
@mgdelacroix It dawned on me that these aren't "gets" they're "filters".

Reply:
Totally! :+1: